src/helpers/postFeedHelpers.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { getHtml } from '../post/Body';
+import BodyShort from '../post/BodyShort';
+import PostFeedEmbed from '../post/PostFeedEmbed';
+import PostModalLink from '../post/PostModalLink';
+
+const STARTWITHPERCENT = 5;
+
+const getPositions = (text) => {
+  const imgPos = text.indexOf('<img');
+  const embedPos = text.indexOf('<iframe');
+  const percentMultiplier = 100 / text.length;
+  const firstEmbed = embedPos !== -1 ? embedPos * percentMultiplier : undefined;
+  const firstImage = imgPos !== -1 ? imgPos * percentMultiplier : undefined;
+  return { embed: firstEmbed, image: firstImage };
+};
+
+const postWithPicture = (tagPositions, searchPostion) => {
+  const result = (tagPositions.image && tagPositions.image < searchPostion);
+  if (tagPositions.embed !== undefined) {
+    return (tagPositions.embed > tagPositions.image) && result;
+  }
+  return result;
+};
+
+const postWithAnEmbed = (tagPositions, searchPostion) => {
+  const result = (tagPositions.embed && tagPositions.embed < searchPostion);
+  if (tagPositions.image !== undefined) {
+    return (tagPositions.image > tagPositions.embed) && result;
+  }
+  return result;
+};
+
+const postStartsWithAPicture = tagPositions => postWithPicture(tagPositions, STARTWITHPERCENT);
```

Comment:
sorry for wrong hints before in my review, I think since these functions are returning boolean should be named like: `isPostStartsWith...`